src/main.cpp

```diff
@@ -4541,7 +4541,7 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                         assert(!"cannot load block from disk");
                     if (inv.type == MSG_CMPCT_BLOCK)
                     {
-                        if (mi->second->nHeight >= chainActive.Height() - 10) {
+                        if (mi->second->nHeight >= chainActive.Height() - MAX_BLOCKS_TO_ANNOUNCE) {
```

Comment:
MAX_BLOCKS_TO_ANNOUNCE (confusing name...) has a very different meaning...I'd prefer to either leave 10 or introduce a new constant.